tensorboard/manager.py

```diff
@@ -300,3 +302,129 @@ def get_all():
     else:
       results.append(info)
   return results
+
+
```

Comment:
I suggest adding a comment here saying that the next three collections represent return codes for the start() function.

Reply:
Can do (though there’s not really a better way than an awkward floating
comment AFAIK—it sure would be nice if the programming language offered
built-in support for defining new data types!).